Dvorak-Right-Handed/KeyboardViewController.swift

```diff
@@ -14,11 +14,19 @@ class KeyboardViewController: UIInputViewController {
     var keyboardHeightConstraint: NSLayoutConstraint?
     
     override func viewDidLoad() {
-        //TODO: use viewwillload?
         super.viewDidLoad()
         // Perform custom UI setup here
+        
+        let kState: String
+        
         let rect = getKeyboardRectFromBounds()
-        let keyboardView = KeyboardView(frame: rect, kvc: self)
+        let size = max(rect.width, rect.height)
+        if size > 1000 {
```

Comment:
Consider Constants?

Reply:
1000 is a literal value. Consider a Constant in stead of the literal value. No big deal though since it is only used here.